README.md

```diff
@@ -5,7 +5,7 @@
 
 First Capstone Project
 
-![image](https://user-images.githubusercontent.com/88060989/144926829-b723e912-7e48-43c6-b0d7-5b343578ae26.png)
+![image](https://user-images.githubusercontent.com/88060989/148611740-8c94fa74-d46e-4b00-97fe-50c8b662cd49.png)
```

Comment:
## [OPTIONAL]
- [ ] One of the requirements says that you need to deploy your project using Github pages and add the live demo link in your README. Let's go ahead and kindly implement this feature 💪 💪